.circleci/config.yml

```diff
@@ -201,6 +201,19 @@ commands:
           fail_only: true
           failure_message: 'Oh no! The **$CIRCLE_JOB** job has failed for **<< parameters.template >>**.\n\n**Relevant PRs of the last 24h:**\n$(git log --merges --since="24 hours ago" --pretty=format:"\`%h\` %<(12)%ar %s [%an]" | grep "Merge pull request" | sed "s/Merge pull request #/https:\/\/github.com\/storybookjs\/storybook\/pull\//g" | tr "\\n" "\\\\n" | sed "s/\\\\/\\\\n/g")'
 jobs:
+  pretty-docs:
+    executor:
+      class: small
+      name: sb_node_16_classic
+    steps:
+      - checkout_advanced:
+          clone_options: '--depth 1 --verbose'
+      - run:
+          name: Prettier
+          command: |
+            cd docs
+            yarn
+            yarn prettier:check
```

Comment:
Let's use the scripts directory for this purpose!
Let's not add another place where we install node-modules.

We could add a npm-script to the root package.json, or a npm-script to the code dir (where we have a bunch of npm-scripts related to linting already)

Reply:
Sounds good to me!

Reply:
And this